rdmo/projects/viewsets.py

```diff
@@ -521,29 +528,33 @@ def dispatch(self, *args, **kwargs):
 
     def retrieve(self, request, *args, **kwargs):
         page = self.get_object()
-        conditions = page.conditions.select_related('source', 'target_option')
-
+        catalog = self.project.catalog
         values = self.project.values.filter(snapshot=None).select_related('attribute', 'option')
 
-        if check_conditions(conditions, values):
+        sets = compute_sets(values)
```

Comment:
I wonder how much more expensive this is. In the future we could move functions like `compute_sets` to the manager. Just having ideas...